src/stdlib/WebSharper.JQuery/Definition.fs

```diff
@@ -706,15 +732,13 @@ module Definition =
             "escapeSelector" => T<string> ^-> T<string>
 
             // Data related static methods
-            "data" => T<Dom.Element> * T<string> * T<obj> ^-> TSelf

```

Comment:
This needs some reworking based on this: 

https://api.jquery.com/jQuery.data/#jQuery-data-element-key-value

The first removed line should be readded with returning T<obj> instead of TSelf

The second removed line shoud be kept as removed

and

```
"data" => T<Dom.Element> * T<obj>
```

should be changed to 

```
"data" => T<Dom.Element> ^-> T<obj>
```